src/screens/stationSelect.tsx

```diff
@@ -95,9 +99,16 @@ const StationSelectScreen: React.FC<StationSelectProps> = ({ navigation }) => {
         value={arriveStationCrsCode}
         setValue={setArriveStationCrsCode}
       />
-      <Button mode="contained" onPress={() => handleButtonTap()}>
-        Submit
-      </Button>
+      <View style={styles.buttonContainer}>
+        <Button mode="contained" onPress={() => handleButtonTap()}>
+          View Journeys
+        </Button>
+      </View>
+      <View style={styles.errorMessageContainer}>
+        {showErrorMessage && errorMessage !== null && (
```

Comment:
As above, no need for both of these and actually more robust and efficient with just one.